frontend/src/components/gameBoard/gameBoard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./gameBoard.module.css";
+import StickInsectDiv from "../stickInsectsContainer/stickInsectContainer";
+
+/**
+ * This component is the Game Board a child component of the Game Page.
+ *
+ * receiving props:
+ *   - TBD
+ *   - Background Image filename
+ *   - Removed for es-lint until can be used
+ */
+export function GameBoard() {
```

Comment:
Since the GameBoard is already default-exported on line 21, you can safely remove the export keyword on this line.